packages/neuron-wallet/src/menu.ts

```diff
@@ -1,21 +1,13 @@
-import { app, shell, Menu, MenuItem, MenuItemConstructorOptions, dialog, BrowserWindow } from 'electron'
-import { Channel, Routes } from './utils/const'
+import { app, Menu, MenuItem, MenuItemConstructorOptions, BrowserWindow } from 'electron'
 import env from './env'
+import dispatch from './commands/dispatcher'
+import Command from './commands/commands'
```

Comment:
In `main.js`, `Command` is imported from `./command/dispatcher`

Reply:
Nice catch.